docs/aggregation-layer/v0.3.5/CompatibilityTable.md

```diff
@@ -0,0 +1,30 @@
+# Compatibility Table
+
+The following compatibility table shows which components should be used depending on the chain you are working with. This table helps ensure that the correct configurations and contracts are selected for each setup:
+
+|    Chain     |  verifierType   |        SC consensus         |  Proofs/Prover   |       Client       | AgglayerBridgeL2 | AgglayerGERL2 + AggOracle | AggOracleCommittee  | AggSender | Provers |
```

Comment:
Maybe adding section `L2 SCs` and under this section: `AgglayerBridgeL2` + `AgglayerGERL2`.
Another section `Components`: `AggOracle` + `Provers`.

Explain one line differences between: `etrog-bridge <> AgglayerBridgeL2` & `etrog-manager-ger <> AgglayerGERL2`